.github/workflows/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] It would be better if we improve our README file as mentioned in the previous code review. Here is the [Microverse Recommended Template](https://github.com/microverseinc/readme-template)